### DISCUSSION TITLE
Finding number of  structures per cluster in rna_cluster

### DISCUSSION THREAD

**Discussion by mandar5335** [RosettaCommons/rosetta#176] [Rosetta]
Hi,

Is there any way to find number of structures per cluster after using `rna_cluster` ?

I can extract cluster representative using `extract_pdbs`, but could not get much more information about each cluster.

Best,
Mandar

**Comment by roccomoretti** [comment]
In the silent file output, there's a `nclust` entry which I believe should be the total number of poses in the corresponding cluster. This may or may not make its way to the information at the bottom of the PDB file when you run `extract_pdbs`.

If not, you can always `grep` the silent file for the `SCORE:` line with the corresponding structure tag, and then attempt to extract the nclust value from the appropriate position in the score line.

**Comment by mandar5335** [comment]
Thanks for the reply, I could get nclust values.

However, I am facing another issue with `rna_cluster`. I am using rna_cluster to cluster top 1% score sorted structures with 5 Angstrom radius.
I am assuming `rna_cluster` sorts by score structures and then clusters top 167 score-sorted structures. 

`rna_cluster.mpi.linuxgccrelease -in:file:silent ../simple_cycle.out -out:file:silent clustered.out -nstruct 167 -cluster:radius 5`

However, all 167 structures are assigned to single cluster. I see RMSD difference is ~ 16 Angstroms in top 2 structures. 

For another Rosetta run, also, I observed similar result. There I get different clusters. But RMSD (calculated with `Pymol`, rechecked with Gromacs's `gmx rms`) is more than 5 A for top  2 structures in the same cluster. 

Is there anything missing in my command line?

**Comment by mandar5335** [comment]
Hi,
RMSD of structures in same cluster is higher than cutoff used for clustering. Any suggestions on rna_cluster RMSD clustering?
